modules/amp_dfp/amp_dfp.module

```diff
@@ -1,4 +1,8 @@
 <?php
+
+include_once 'includes/amp_rtc_config.admin.inc';
```

Comment:
Do you need this include? Should not the form do that in 'file' => 'includes/amp_rtc_config.admin.inc'?